src/dex/layouts/CardGridLayout.tsx

```diff
@@ -38,7 +37,17 @@ export function CardGridLayout<QueryResultDataType>(props: CardGridLayoutProps<Q
   }
 
   if (queryResult.isLoading) {
-    return <LoadingSpinnerLayout />;
+    // return <LoadingSpinnerLayout />;
+    return (
```

Comment:
I'm seeing the error and loading state appear at the same time:
![Screen Shot 2023-10-26 at 10 11 01 AM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/8b446e60-b510-4446-a90c-b3f540c3f729)

Reply:
Done.